VirtoCommerce.DynamicExpressionsModule.Data/Promotion/Conditions/CartConditions/ConditionAtCartItemExtendedTotal.cs

```diff
@@ -1,38 +1,51 @@
-﻿using System;
+using System;
 using VirtoCommerce.Domain.Common;
 using VirtoCommerce.Domain.Marketing.Model;
 using linq = System.Linq.Expressions;
 
 namespace VirtoCommerce.DynamicExpressionsModule.Data.Promotion
 {
-	//Line item subtotal is []
-	public class ConditionAtCartItemExtendedTotal : ConditionBase, IConditionExpression
-	{
-		public decimal LineItemTotal { get; set; }
-
-		public bool Exactly { get; set; }
-		
-		#region IConditionExpression Members
-		/// <summary>
-		/// ((PromotionEvaluationContext)x).IsAnyLineItemTotal(LineItemTotal, Exactly,  ExcludingCategoryIds, ExcludingProductIds)
-		/// </summary>
-		/// <returns></returns>
-		public linq.Expression<Func<IEvaluationContext, bool>> GetConditionExpression()
-		{
-			var paramX = linq.Expression.Parameter(typeof(IEvaluationContext), "x");
-			var castOp = linq.Expression.MakeUnary(linq.ExpressionType.Convert, paramX, typeof(PromotionEvaluationContext));
-			var lineItemTotal = linq.Expression.Constant(LineItemTotal);
-			var methodInfo = typeof(PromotionEvaluationContextExtension).GetMethod("IsAnyLineItemExtendedTotal");
-
-			var equalsOrAtLeast = Exactly ? linq.Expression.Constant(true) : linq.Expression.Constant(false);
-			var methodCall = linq.Expression.Call(null, methodInfo, castOp, lineItemTotal, equalsOrAtLeast, GetNewArrayExpression(ExcludingCategoryIds),
-																	  GetNewArrayExpression(ExcludingProductIds));
-
-			var retVal = linq.Expression.Lambda<Func<IEvaluationContext, bool>>(methodCall, paramX);
-
-			return retVal;
-		}
-
-		#endregion
-	}
-}
\ No newline at end of file
+    //Line item subtotal is []
+    public class ConditionAtCartItemExtendedTotal : ConditionBase, IConditionExpression
+    {
+        public decimal LineItemTotal { get; set; }
+
+        public decimal LineItemTotalSecond { get; set; }
+
+        public string CompareCondition { get; set; }
```

Comment:
Won't Enum make us deal with integer values instead of strings, which are easy-to-understand? The "string" idea was taken from "CompareConditionBase.cs" where the similar multi-optional conditional expression is built. Also the 'compareConditionToText' filter for string values is already implemented, we just need to add some values.